examples/api/lib/widgets/sliver/sliver_tree.0.dart

```diff
@@ -0,0 +1,104 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [SliverTreeList].
+
+void main() => runApp(const SliverTreeExampleApp());
+
+class SliverTreeExampleApp extends StatelessWidget {
+  const SliverTreeExampleApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      home: SliverTreeExample(),
+    );
+  }
+}
+
+class SliverTreeExample extends StatefulWidget {
+  const SliverTreeExample({super.key});
+
+  @override
+  State<SliverTreeExample> createState() => _SliverTreeExampleState();
+}
+
+class _SliverTreeExampleState extends State<SliverTreeExample> {
+  SliverTreeNode<String>? _selectedNode;
+  final SliverTreeController controller = SliverTreeController();
+  final List<SliverTreeNode<String>> _tree = <SliverTreeNode<String>>[
+    SliverTreeNode<String>('First'),
+    SliverTreeNode<String>(
+      'Second',
+      children: <SliverTreeNode<String>>[
+        SliverTreeNode<String>(
+          'alpha',
+          children: <SliverTreeNode<String>>[
+            SliverTreeNode<String>('uno'),
+            SliverTreeNode<String>('dos'),
+            SliverTreeNode<String>('tres'),
+          ],
+        ),
+        SliverTreeNode<String>('beta'),
+        SliverTreeNode<String>('kappa'),
+      ],
+    ),
+    SliverTreeNode<String>(
+      'Third',
+      expanded: true,
+      children: <SliverTreeNode<String>>[
+        SliverTreeNode<String>('gamma'),
+        SliverTreeNode<String>('delta'),
+        SliverTreeNode<String>('epsilon'),
+      ],
+    ),
+    SliverTreeNode<String>('Fourth'),
+  ];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('SliverTree demo'),
+      ),
+      body: CustomScrollView(
+        slivers: <Widget>[
+          SliverTreeList<String>(
+            tree: _tree,
+            controller: controller,
+            treeNodeBuilder: (
```

Comment:
Is there a way to customize the padding between the nodes and the labels? 
It looks like gap between the nodes and labels is bit when you see something like GitHub or VS Code tree list.

Preview 1  | Preview 2 | Preview 3 |
| --------------- | --------------- |--------------- |
| <img src="https://github.com/flutter/flutter/assets/48603081/e3c07388-d696-493b-827a-e7634335061b" > |  <img src="https://github.com/flutter/flutter/assets/48603081/a1b8d208-c8c1-493b-ae02-ed41398622b8" > | <img src="https://github.com/flutter/flutter/assets/48603081/fe6ade2a-280f-4fb2-9a46-d772748864c6" > |

Reply:
Yes, same as TreeView, there is a default builder users can customize.